packages/nx/src/command-line/init/init-v2.ts

```diff
@@ -51,18 +54,47 @@ export function installPlugins(
   runInstall(repoRoot, pmc);
 
   output.log({ title: '🔨 Configuring plugins' });
+  const succeededPlugins = [];
+  const failedPlugins = [];
   for (const plugin of plugins) {
-    execSync(
-      `${pmc.exec} nx g ${plugin}:init --keepExistingVersions ${
-        updatePackageScripts ? '--updatePackageScripts' : ''
-      }`,
-      {
-        stdio: [0, 1, 2],
-        cwd: repoRoot,
-        windowsHide: false,
-      }
-    );
+    try {
+      execSync(
+        `${pmc.exec} nx g ${plugin}:init --keepExistingVersions ${
+          updatePackageScripts ? '--updatePackageScripts' : ''
+        }`,
+        {
+          stdio: [0, 1, 2],
+          cwd: repoRoot,
+          windowsHide: false,
+        }
+      );
+      succeededPlugins.push(plugin);
+    } catch (e) {
+      failedPlugins.push(plugin);
+      output.error({
+        title: `Install plugin ${plugin} failed: ${
+          e.message || 'Unknown error'
+        }`,
+        bodyLines: [e.stack],
+      });
+    }
+  }
+  if (succeededPlugins.length > 0) {
+    output.success({
+      title: 'Installed Plugins',
+      bodyLines: succeededPlugins.map((p) => `- ${p}`),
+    });
+  }
+  if (failedPlugins.length > 0) {
+    output.error({
+      title: 'Failed to install Plugins',
+      bodyLines: [
+        ...failedPlugins.map((p) => `- ${p}`),
+        'You may need to install them manually.',
+      ],
+    });
```

Comment:
So what is the state after this?

The failed plugins will have been `npm` installed.. but they won't be added to `nx.json` right?

Can we show the error details here?

Installing the plugin manually would result in the same thing right? Are there cases where it will suddenly succeed after it just failed? We should instruct people to do something which will fix the issue...

Reply:
i changed something like:
<img width="1079" alt="Screenshot 2024-11-08 at 5 37 16 PM" src="https://github.com/user-attachments/assets/3f5fae8f-3173-498b-92de-4b7eaa329e30">